crates/pop-cli/tests/contract.rs

```diff
@@ -67,13 +67,24 @@ async fn contract_lifecycle() -> Result<()> {
 		.success();
 	assert!(temp_dir.join("test_contract").exists());
 
-	// pop build --path ./test_contract --release
+	/*	// pop build --path ./test_contract --release
```

Comment:
I'd keep this code and remove the one below, test it with the `--path` flag in this specific case